### PR TITLE
Sve4a auth

### PR DESCRIPTION
**Pull request by sve4a**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,6 +24,5 @@ yarn-debug.log*
 yarn-error.log*
 
 #credentials
-/.env
```
Comment (antilik) [inline .gitignore]: it's necessary to leave it here to prevent downloading mentioned file.

Comment (sve4a) [reply .gitignore]: Slash from start means that  file is global. So that's not necessary =)

File: client/src/components/Auth/LoginForm.js [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+                input: classes.input,
+              }}
+              value={state.password}
+              onChange={onChange('password')}
+              labelWidth={105}
+              fullWidth
+              endAdornment={
+                state.password
+                  ? (
+                    <PasswordIcon
```
Comment (antilik) [inline client/src/components/Auth/LoginForm.js]: password icon has to be with the property (cursor: pointer;).
visibility of password is broken with resolution lower than 900px.

File: client/src/components/Auth/RegisterForm.js [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+              }}
+              value={state.password}
+              onChange={onChange('password')}
+              labelWidth={75}
+              fullWidth
+              endAdornment={
+                state.password
+                  ? (
+                    <PasswordIcon
+                      onMouseDown={() => setPasswordVisible(false)}
```
Comment (antilik) [inline client/src/components/Auth/RegisterForm.js]: the same things as was mention in LoginForm.

File: client/src/components/Auth/LoginForm.js (hunk 2 above)
Comment (sve4a) [reply client/src/components/Auth/LoginForm.js]: Resolved

File: .gitignore (hunk 1 above)
Comment (sve4a) [reply .gitignore]: Resolved

File: client/src/components/Auth/RegisterForm.js [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+    setState({ ...state, [stateName]: event.target.value });
+  };
+
+  const onSubmit = (event) => {
+    event.preventDefault();
+
+    const { email, login, password } = state;
+
+    if (validate()) {
+      console.log(state, 'REQUEST');
```
Comment (antilik) [inline client/src/components/Auth/RegisterForm.js]: it's better to comment console.log or delete.

File: client/src/components/Auth/LoginForm.js [hunk 5]
```diff
@@ -0,0 +1,338 @@
...
+const museo = {
+  fontFamily: 'Museo Sans 500',
+  fontStyle: 'normal',
+  src: `
+    local('Museo Sans 500'),
+    url(${MuseoSans}) format('woff')
+  `,
+};
+
+const useMobileStyles = makeStyles((theme) => ({
```
[the remark this answers is not in the document]
Comment (sve4a) [reply client/src/components/Auth/LoginForm.js]: Do you know horizontal devices with 900px width?

File: client/src/components/Auth/RegisterForm.js (hunk 3 above)
Comment (sve4a) [reply client/src/components/Auth/RegisterForm.js]: Resolved

File: client/src/components/Auth/RegisterForm.js (hunk 4 above)
Comment (sve4a) [reply client/src/components/Auth/RegisterForm.js]: resolved

File: client/src/components/Auth/LoginForm.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (antilik) [reply client/src/components/Auth/LoginForm.js]: I wrote and lower. Pls look at the comment. The line exists with 768 resolution.

[the remark this answers is not in the document]
Comment (antilik) [reply client/src/components/Auth/LoginForm.js]: this fix will be uploaded later.

**Pull request merged**